src/org/opensolaris/opengrok/configuration/Project.java

```diff
@@ -229,6 +234,20 @@ public void setNavigateWindowEnabled(boolean navigateWindowEnabled) {
         this.navigateWindowEnabled = navigateWindowEnabled;
     }
 
+    /**
+     * @return true if this project handles renamed files.
+     */
+    public boolean isHandleRenamedFiles() {
+        return handleRenamedFiles;
```

Comment:
this cause potential NPE when `handleRenamedFiles` is null, doesn't it?

Reply:
yes, did not mean to expose `null` outside.